spec/rubocop/cop/style/access_modifier_declarations_spec.rb

```diff
@@ -633,11 +633,13 @@ def bar; end
           class Test
             %{access_modifier}
             ^{access_modifier} `#{access_modifier}` should be inlined in method definitions.
+            def foo; end
```

Comment:
Please add new test instead of modifying existing test.

Reply:
This test should be modified since it asserts that this code leads to an offense:

```ruby
class Test
  private
end
```

the whole point of this PR is to omit offense in this case